### PR TITLE
Fix minor warnings in xunit tests

### PR DESCRIPTION
**Pull request by zvirja**
Fix a few minor issues in our unit tests:
1. Use valid assertion method.
2. Remove duplicated theory data. One more confirmation that copy-paste approach is error prone 😉

@moodmosaic @adamchester Please review and accept - changes are very tiny.

### CODE REVIEW

File: Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs [hunk 1]
```diff
@@ -125,16 +124,12 @@ public void CreateWithRangeAttributeRequestReturnsCorrectResult(Type type, objec
         [InlineData("Property", "-2.2", "-1.1")]
         [InlineData("Property", 10.0, 20.0)]
         [InlineData("Property", -2.0, -1.0)]
-        [InlineData("Property", 10, 20)]
```
Comment (moodmosaic) [inline Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs]: Why are those removed?

Comment (zvirja) [reply Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs]: If you look around you'll notice that are duplicated 😉 You are the author of these lines so it seems that you mistakenly introduced that by copy-paste 😄

Comment (zvirja) [reply Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs]: image

Comment (moodmosaic) [reply Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs]: 👍 👍 😄
